packages/admin-panel/src/VizBuilderApp/components/PreviewSection.jsx

```diff
@@ -186,9 +259,9 @@ export const PreviewSection = () => {
         <TableContainer>
           {showData ? (
             <FetchLoader
-              isLoading={isLoading || isFetching}
-              isError={isError}
-              error={error}
+              isLoading={isTableLoading || isTableFetching}
```

Comment:
```suggestion
              isLoading={isTableFetching}
```

I believe `isLoading` implies `isFetching`. Ditto line 280